### PR TITLE
Fix parallel tests unreliably failing due to statics in Atmospherics

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Fixes https://pastebin.com/vedx6kxq
Fixes https://pastebin.com/zhN6RD24

### CODE REVIEW

File: Content.Server/Atmos/GasMixture.cs [hunk 1]
```diff
@@ -48,10 +50,11 @@ public float HeatCapacity
             get
             {
                 var capacity = 0f;
+                var atmosSystem = EntitySystem.Get<AtmosphereSystem>();
```
Comment (Zumorica) [inline Content.Server/Atmos/GasMixture.cs:53]: No. I refuse to let this be here.

File: Content.Server/Atmos/GasMixture.cs [hunk 2]
```diff
@@ -65,10 +68,11 @@ public float HeatCapacityArchived
...
```
Comment (Zumorica) [inline Content.Server/Atmos/GasMixture.cs:71]: Same here.

**Comment by DrSmugleaf** [comment]
@Zumorica requesting a hell review

File: Content.Server/Atmos/GasMixture.cs [hunk 3]
```diff
@@ -265,6 +269,7 @@ public float Share(GasMixture sharer, int atmosAdjacentTurfs)
             var heatCapacitySharerToThis = 0f;
             var movedMoles = 0f;
             var absMovedMoles = 0f;
+            var atmosSystem = EntitySystem.Get<AtmosphereSystem>();
```
Comment (Zumorica) [inline Content.Server/Atmos/GasMixture.cs:272]: Same.

File: Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs [hunk 4]
```diff
@@ -161,11 +162,12 @@ private bool TryRefreshTile(GridAtmosphereComponent gam, GasOverlayData oldTile,
             }
 
             var tileData = new List<GasData>();
+            var atmosSystem = Get<AtmosphereSystem>();
```
Comment (Zumorica) [inline Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs:165]: I'd rather you cache this.

**Review by Zumorica — CHANGES_REQUESTED** [review]

File: Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs (hunk 4 above)
Comment (metalgearsloth) [inline Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs:165]: Can you just store this as a member variable on GasTileOverlaySystem? TryRefreshTile is called a lot.

File: Content.Server/Atmos/GasMixture.cs (hunk 1 above)
Comment (metalgearsloth) [inline Content.Server/Atmos/GasMixture.cs:53]: From memory when I profiled using this variable here it adds on a bit to the perf but fixing the parallelisation is more important.

File: Content.Server/Atmos/GasMixture.cs [hunk 5]
```diff
@@ -22,12 +22,17 @@ namespace Content.Server.Atmos
     [Serializable]
     public class GasMixture : IExposeData, IEquatable<GasMixture>, ICloneable
     {
+        private readonly AtmosphereSystem _atmosphereSystem;
```
Comment (Zumorica) [inline Content.Server/Atmos/GasMixture.cs:25]: Although this makes me terribly sad, I cannot disapprove. :pensive:

**Review by Zumorica — APPROVED** [review]

**Pull request merged**